submitted_models/marble_spot_sensor_config_1/model.config

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0"?>
 <model>
-  <name>ctu_cras_norlab_spot_sensor_config_3</name>
-  <sdf version="1.7">model.sdf</sdf>
+  <name>ctu_cras_norlab_spot_sensor_config_1</name>
```

Comment:
Should this be `marble_spot_sensor_config_1`?

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d